datasets/us/plum_book/crawler.py

```diff
@@ -0,0 +1,54 @@
+import csv
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+
+IGNORE = [
+    "OrganizationName",
+    "PositionStatus",
+    "PaymentPlanDescription",
+    "LevelGradePay",
+    "Tenure",
+]
+
+
+def crawl(context: Context):
+    path = context.fetch_resource("source.csv", context.data_url)
+    with open(path, "r", encoding="utf-8") as fh:
+        reader = csv.DictReader(fh)
+        for row in reader:
+            agency_name = row.pop("\ufeffAgencyName")
+            position_title = row.pop("PositionTitle", None)
+            appointment_type = row.pop("AppointmentTypeDescription")
+            expiration_date = row.pop("ExpirationDate")
+            location = row.pop("Location")
+            incumbent_first_name = row.pop("IncumbentFirstName")
+            incumbent_last_name = row.pop("IncumbentLastName")
+
+            if not incumbent_first_name or not incumbent_last_name:
+                continue
+            person = context.make("Person")
+            person.id = context.make_slug(incumbent_first_name, incumbent_last_name)
```

Comment:
It sucks, but I think we need to put more information into the ID. I think the best thing to add is position_name, since so many positions are going to be in washington, and there's just a really good chance of the same name for two individuals coming up.

```
grep '"Occupancy"' data/datasets/us_plum_book/entities.ftm.json | jq .properties.holder[0]|sort|uniq -c|sort -n
...
   2 "plum-thomas-sullivan"
   2 "plum-william-brown"
   2 "plum-william-harris"
   3 "plum-michael-horowitz"
```